semantic_conventions/metrics/process-runtime-jvm-metrics.yaml

```diff
@@ -63,8 +64,9 @@ groups:
     brief: "Duration of JVM garbage collection actions."
     instrument: histogram
     unit: "s"
+    prefix: gc
```

Comment:
I'm not sure if using the prefix for the metric itself is a good idea. Also, `gc` might be too broad. If we want to have other "gc" attributes, then they will collide with the ones here for this metric.

Reply:
I thought this was just a tool to make the yaml file shorter, nothing that is changing the spec in any way.
If this is not true, then we may need to take this issue back to discussion.

Reply:
that was not a clear yes or no answer - I'm a bit lost

Reply:
It's not changing the spec, but it's kinda opening up for problems later I think. I expressed it in more detail [here](https://github.com/open-telemetry/semantic-conventions/pull/20#issuecomment-1610977656), mostly aligns with the concerns from @jack-berg I think. 

I guess we need to think about what the `memory` or `gc` namespaces will contain and if the attributes here are "generic" enough so that they can be used somewhere else. OR we pick a more specific namespace for them, like `jvm.gc`

Reply:
yes, that was very helpful :smile: